src/Models/EventAttendee.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace TruckersMP\APIClient\Models;
+
+use Carbon\Carbon;
+
+class EventAttendee
+{
+    /**
+     * The attendee's user ID.
+     *
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * The attendee's username.
+     *
+     * @var string
+     */
+    protected $username;
+
+    /**
+     * If the attendee is being followed.
+     * TODO: Check what this field means/does.
```

Comment:
@ShawnCZek are you able to provide a bit more information on what this value is actually used for?

Reply:
The field is currently unused and is reserved for the upcoming features.  
With that being said, it can now be supported as there should not be any additional changes.

Reply:
Sounds good.

@ShawnCZek what description do you want me to add to the DocBlock? Or is the current fine.

Reply:
It will tell whether the person is following news (changes) of the event.